src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -135,7 +130,9 @@ List only the disabled partitions.
 		},
 	}
 	cmd.Flags().BoolVarP(&all, "all", "a", false, "If true, list all partitions in the cluster")
+	cmd.Flags().MarkHidden("all")
```

Comment:
Could just outright remove this flag in the PR, rather than mark it hidden (same w/ disabled-only below)

Reply:
Although I guess for "stability" purposes since this is officially released, MarkHidden works as well.